packages/nimble-components/src/anchor-tabs/specs/README.md

```diff
@@ -0,0 +1,204 @@
+# Anchor Tabs
+
+## Overview
+
+Anchor tabs are a sequence of tabs that navigate to URLs when activated.
+
+### Background
+
+[Visual Design spec](https://xd.adobe.com/view/33ffad4a-eb2c-4241-b8c5-ebfff1faf6f6-66ac/screen/b2aa2c0c-03b7-4571-8e0d-de88baf0814b/specs/) (same visual design as non-anchor tabs)
+
+[GitHub issue](https://github.com/ni/nimble/issues/479)
+
+### Non-goals
+
+- Mixing regular tabs and anchor tabs together
+- Vertical tabs orientation
+
+### Features
+
+- Functions like an anchor (i.e. navigates) upon activation
+- Supports keyboard navigation within group of tabs without automatically activating each focused tab
+- Disable individual tabs
+
+### Risks and Challenges
+
+None
+
+### Prior Art/Examples
+
+In SLE:
+
+![SLE](sle-systems.png)
+
+In GitHub:
+
+![GitHub](github.png)
+
+---
+
+## Design
+
+Anchor tabs have the same appearance as a standard tabs control, i.e. a sequence of operable regions with labels, each of which corresponds to some target content to display. In the case of anchor tabs, they do not switch between pre-configured tab panels, but rather navigate to some URL. The tabs can be configured to load the URL in a specific target, like an `iframe`.
+
+The design consists of an "Anchor tabs" container component which contains one or more "Anchor tab" components. This mirrors the existing design of the `nimble-tabs` and `nimble-tab` components, however there is no analog for the `nimble-tab-panel` component. Instead, the content loaded from the configured URL(s) loads into the containing page, an `iframe`, a new window, etc.
+
+We cannot extend FAST's tabs control because it has baked into it the idea that tabs activate tab panels. Instead we will create new components from scratch, but we can likely borrow logic from FAST e.g. for handling keyboard navigation.
+
+### API
+
+#### Anchor Tabs
+
+- *Component Name* - `nimble-anchor-tabs`
+- *Props/Attrs*
+    -   `activeid` - id of active tab. Defaults to the first tab.
+- *Methods* - None
+- *Events* - None
```

Comment:
I guess so? Are there guidelines for what events should be provided for a given control? Anyway, I've added the change event.

Reply:
I actually struggled with this and I'm not sure we should. When a user clicks / activates a new tab they are performing a navigation, not activating a tab behavior. I'm currently thinking it should be the apps responsibility to render the page based on the URL state and not the tab control itself.

So I think I don't expect the activeId to change or for a change event to be fired because we could end up fighting the application's router. ie what if a RouterGuard blocks the navigation? if the control had immediately assumed that a click would activate the next tab then the UI would be out of sync because the RouterGuard blocked the nav.

The app already has to do work to select the correct tab as active based on the route, they should also have to do the work of updating the active tab based on the route after navigation.

Another phrasing: If the app dev wants different behavior based on the "tab changing" then they want a normal tab control, not anchor-tabs. Anchor-tabs are just stylized links.

This thought process also makes me think we would not do the following:

> The `Enter` or `Space` key will activate the focused tab, setting `aria-selected`, moving the active indicator, and loading the tab's associated URL.

The enter or space key will only load the tab's associated url (because the user is activating a link). And the "new page" is responsible for updating what is the activeid.

Reply:
I'll change the spec to make it clear that the active tab state is independent of any navigation that actually occurs, and that it is up to clients to keep the active tab in sync with what is loaded.

As far as activating with the `Space` key, I think we should continue to do that, since that is part of the expected keyboard interactions for `tablist`/`tab` roles.